roles/confluent.kerberos/defaults/main.yml

```diff
@@ -14,3 +14,19 @@ kerberos:
   default_tkt_enctypes: aes256-cts-hmac-sha1-96 aes128-cts-hmac-sha1-96 arc-four-hmac rc4-hmac
   default_tgs_enctypes: aes256-cts-hmac-sha1-96 aes128-cts-hmac-sha1-96 arc-four-hmac rc4-hmac
   permitted_enctypes: aes256-cts-hmac-sha1-96 aes128-cts-hmac-sha1-96 arc-four-hmac rc4-hmac
+  # Deprecated Vars
+  realm: ""
+  kdc_hostname: ""
+  admin_hostname: ""
+
+### Boolean to configure Kerberos krb5.conf file, must also set kerberos_realm, keberos_kdc_hostname, kerberos_admin_hostname
+kerberos_configure: true
+
+### KDC Server Realm
+kerberos_realm: "{{ kerberos.realm }}"
+
+### KDC Server Hostname
+keberos_kdc_hostname: "{{ kerberos.kdc_hostname }}"
+
+### KDC Admin Server Hostname
```

Comment:
can this var just be set to keberos_kdc_hostname? ppl will have the option to set both but most cases only need to set one

Reply:
should we really be deprecating this variable? will at least need to update the molecule scenarios

Reply:
I don't think we should deprecate this, as if someone is running MIT KDC they will need it.  We could set them to be the same as 90% of the time you will want that configuration, as long we leave an option to override.

Reply:
im going to put this change in a different pr, it touches too many files. But I think its a good idea, ill turn this into a todo

Reply:
Sounds good.